RTNeural/gru/gru.h

```diff
@@ -85,7 +85,7 @@ class GRULayer final : public Layer<T>
 
         T** W;
         T** U;
-        T* b[2];
+        T** b;
```

Comment:
My only concern here is that the compiler can't optimize the layer operation as well when dynamic allocation is used here. From looking at the [benchmarks results](https://github.com/jatinchowdhury18/RTNeural/pull/29/checks?check_run_id=3063432186#step:8:7) from your PR though, I think it will be okay. That said, running benchmarks through GitHub Actions isn't super reliable, so you may want to run the benchmarks locally before and after this change to confirm.

Reply:
The benchmark results seem pretty close to the benchmark results before this change when I ran it locally. :)